source/includes/steps-import.yaml

```diff
@@ -19,25 +21,61 @@ content: |
 
   .. figure:: /images/compass/import-data-dialog.png
      :alt: Import data dialog
+     :width: 60%
 
 ---
-title: Choose the appropriate file type of the source data file. 
+title: Select the location of the source data file under 
+       :guilabel:`Select File`.
+level: 4
+ref: click-file-location
+content: ""
+
+---
+title: Choose the appropriate file type.
 level: 4
 ref: choose-file-type
 content: |
-  Under :guilabel:`Select Input File Type`, select either JSON or CSV. 
+
+  Under :guilabel:`Select Input File Type`, select either 
+  :guilabel:`JSON` or :guilabel:`CSV`. 
+
+  If you are importing a CSV file, you may specify fields to import and 
+  the types of those fields under :guilabel:`Specify Fields and Types`. By default, |compass-short| imports every field as a string.
```

Comment:
Wrap here. Also, I think we can tighten up the last sentence a bit: "The default data type for all fields is string".